scripts/utils/train_helper.py

```diff
@@ -91,8 +143,10 @@ def get_fcp_trained_w_selfplay_types(args,
                                                                   num_self_play_agents_to_train=num_self_play_agents_to_train,
                                                                   parallel=parallel)
     name = 'fcp_w_selfplay_types'
+
     agents = load_agents(args, name=name, tag=tag, force_training=fcp_w_sp_force_training)
     if agents:
+        # TODO: instead of returning here, shouldn't we be using the loaded agent to initialize the fcp_trainer below?
```

Comment:
If it was able to load the fcp_w_sp_types from file then we no longer have to train it to get it